README.md

```diff
@@ -40,7 +41,49 @@ In the examples folder you can find out how to run a Service Node in a browser t
 
 ## API Docs
 
-https://ipfs.io/ipfs/QmYjfSPRcqKeRiJ7Es1qfnfgADSAvM1CApJyZdXhfpDqjs
+https://ipfs.io/ipfs/QmTHQog37CpVHrdwWt1EpS8feymig8N2RpoTBY66P9D8cE
+
+### Events
```

Comment:
Uhm I agree we should choose a rigid scheme. My thinking atm is that we try to keep event names relevant to what causes it and not the consequence. This is so that events are reusable, for example the events about pub keys are emitted when a pub key is discovered in a valid review record, they make no assumptions about the consequences